### PR TITLE
Feature/custom object search create

### PR DESCRIPTION
**Pull request by reese-mitchell**

### CODE REVIEW

File: EloquaClient.Tests/Unit/EloquaContextTests.cs [hunk 1]
```diff
@@ -139,5 +141,98 @@ public async Task DbSetUpdateCustomCampaignObjectAsyncFailTest()
...
+
+            //Assert
+            Assert.IsInstanceOfType(result, typeof(List<CustomObjectData>));
+
+            
+            Assert.AreEqual(10, result[0].Id);
+        }
+        #endregion
+
+        #region SearchCustomCampaignObjectAsync
```
Comment (antwonthegreat) [inline EloquaClient.Tests/Unit/EloquaContextTests.cs]: rename region to CreateCustomCampaignObjectsAsync ?

**Review by antwonthegreat — CHANGES_REQUESTED** [review]

File: EloquaClient/EloquaContext.cs [hunk 2]
```diff
@@ -102,9 +99,66 @@ public async Task<Result> UpdateCustomCampaignObjectsAsync(int state, long custo
...
+
+            var request = new RestRequest(requestUrl, Method.GET);
+
+            var response = await _restClient.ExecuteTaskAsync(request);
+
+            var resultObject = JsonConvert.DeserializeObject<Element<CustomObjectData>>(response.Content);
+
+            var results = new List<CustomObjectData>();
+
+            foreach (var element in resultObject.Elements)
```
Comment (AbandonedFridge) [inline EloquaClient/EloquaContext.cs]: Couldn't we just return resultObject.Elements here?

**Review by AbandonedFridge — CHANGES_REQUESTED** [review]

File: EloquaClient/EloquaContext.cs (hunk 2 above)
Comment (reese-mitchell) [reply EloquaClient/EloquaContext.cs]: Yup,  Didn't realize that Elements was already a list.  Should be removed now

File: EloquaClient.Tests/Unit/EloquaContextTests.cs (hunk 1 above)
Comment (reese-mitchell) [reply EloquaClient.Tests/Unit/EloquaContextTests.cs]: Should be fixed.  It was a copy error...  :-/

**Review by AbandonedFridge — APPROVED** [review]

**Review by antwonthegreat — APPROVED** [review]

**Review by ShawnJohnson14 — APPROVED** [review]

**Pull request merged**